src/interpolation/interpolate.test.js

```diff
@@ -53,6 +54,189 @@ describe('interpolate', () => {
     expect(result).toMatchSnapshot()
     expect(updates).toMatchSnapshot()
   })
+
+  test('full document — strict: Unknown formatter', () => {
+    const document = {
```

Comment:
Can't we reuse the same document instead of rewriting it on every test?

Reply:
Yes, we can.

Reply:
Actually we can't as they are not the same

Reply:
Alright, we can keep it then